pyproject.toml

```
@@ -0,0 +1,104 @@
+[build-system]
+requires = ["setuptools>=61.0", "wheel", "fonttools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "fpdf2"
+dynamic = ["version"]
+description = "Simple & fast PDF generation for Python"
+readme = "README.md"
+requires-python = ">=3.9"
+license = { file = "LICENSE" }
+
+authors = [
+  { name = "Olivier PLATHEY" },
+  { name = "Max" }
+]
+
+maintainers = [
+  { name = "Lucas Cimon" }
+]
+
+keywords = ["pdf", "unicode", "png", "jpg", "ttf", "barcode"]
```

```suggestion
keywords = ["pdf", "unicode", "png", "jpg", "ttf", "barcode", "library", "markdown"]
```
I had a loot at the top 100 most used Pypi keywords: https://chezsoi.org/shaarli/shaare/ZNiBqA

`library` & `markdown` seem like good additions 🙂